test/integration_test.go

```diff
@@ -231,8 +231,7 @@ func TestTerragruntInitHookNoSourceNoBackend(t *testing.T) {
 		t.Errorf("Did not expect to get error: %s", err.Error())
 	}
 
-	// `init` hook should execute only once (2 occurrences due to the echo and its output)
-	assert.Equal(t, 2, strings.Count(output, "AFTER_INIT_ONLY_ONCE"), "Hooks on init command executed more than once")
+	assert.Equal(t, 1, strings.Count(output, "AFTER_INIT_ONLY_ONCE"), "Hooks on init command executed more than once")
```

Comment:
Why change this down to 1?

Reply:
Because this is what the [test expects](https://github.com/gruntwork-io/terragrunt/blob/master/test/fixture-hooks/init-once/no-source-no-backend/terragrunt.hcl#L10)

Reply:
I think seeing it twice was some kind of side effect, which got fixed with this transition.

Reply:
Right, but why was the test originally looking for 2? I'm prob just missing some context, and the comment that was there about "echo and its output" isn't making sense to me.

Reply:
@brikis98 so on `master` the output is this:
```
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] [terragrunt] [/var/folders/96/29zyl8fn0w10t_3y3cc7hl_r0000gn/T/terragrunt-test251996998/fixture-hooks/init-once/no-source-no-backend] 2021/01/28 16:09:45 Detected 2 Hooks
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] [terragrunt] [/var/folders/96/29zyl8fn0w10t_3y3cc7hl_r0000gn/T/terragrunt-test251996998/fixture-hooks/init-once/no-source-no-backend] 2021/01/28 16:09:45 Executing hook: after_init
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] [terragrunt] [/var/folders/96/29zyl8fn0w10t_3y3cc7hl_r0000gn/T/terragrunt-test251996998/fixture-hooks/init-once/no-source-no-backend] 2021/01/28 16:09:45 Running command: echo AFTER_INIT_ONLY_ONCE
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] AFTER_INIT_ONLY_ONCE
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] [terragrunt] 2021/01/28 16:09:45 Running command: terraform apply -auto-approve
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend] [terragrunt] 2021/01/28 16:09:47 Detected 2 Hooks
    integration_test.go:3114: [TestTerragruntInitHookNoSourceNoBackend]
```
First we see `Running command: echo AFTER_INIT_ONLY_ONCE`, and then actual output `AFTER_INIT_ONLY_ONCE`. With this PR `Running command: ...` is not shown by default (will be seeing with debug level). So that is why we see exactly one `AFTER_INIT_ONLY_ONCE` now

Reply:
We can switch `Running command:` to be shown by default (move it from Infof to Warnf), and then the test will be failing with 1 and passing with 2.

Reply:
Ohhhh, I get it! OK, I think the logging improvement here is better, so it's fine to update the test to 1. I just wanted to be sure we understood _why_ that was changing so we could be confident we weren't covering up a bug. I think we're good on this one, thx for digging in 👍